cpp/src/components/rmmAllocatorAdapter.hpp

```diff
@@ -1,96 +0,0 @@
-/*
```

Comment:
How about ` cuml_allocator.hpp`?

```

./cpp/src/components/cuml_allocator.hpp:104:/** Default cudaMalloc/cudaFree based device allocator */
./cpp/src/components/cuml_allocator.hpp:110:    CUDA_CHECK(cudaMalloc(&ptr, n));
./cpp/src/components/cuml_allocator.hpp:125:/** Default cudaMallocHost/cudaFreeHost based host allocator */
./cpp/src/components/cuml_allocator.hpp:131:    CUDA_CHECK(cudaMallocHost(&ptr, n));

```

Reply:
I left this alone because those lines belong to the default allocator. Perhaps that whole file can be removed since MLCommon::deviceAllocator is not being used anywhere?

Reply:
ok, let's give it a try

Reply:
done